dev/integration_tests/ios_add2app/build_and_test.sh

```diff
@@ -5,13 +5,7 @@ set -e
 cd "$(dirname "$0")"
 
 pushd flutterapp
-../../../../bin/flutter build ios --debug --no-codesign -v
-
-pushd .ios
-xcodebuild -workspace Runner.xcworkspace -scheme Runner -destination generic/platform=iOS -configuration Debug CODE_SIGNING_REQUIRED=NO CODE_SIGNING_IDENTITY="" CODE_SIGNING_ALLOWED=NO
-xcodebuild -workspace Runner.xcworkspace -scheme Runner -destination generic/platform=iOS -configuration Profile CODE_SIGNING_REQUIRED=NO CODE_SIGNING_IDENTITY="" CODE_SIGNING_ALLOWED=NO
-xcodebuild -workspace Runner.xcworkspace -scheme Runner -destination generic/platform=iOS -configuration Release CODE_SIGNING_REQUIRED=NO CODE_SIGNING_IDENTITY="" CODE_SIGNING_ALLOWED=NO
-popd
+../../../../bin/flutter build ios --debug --simulator --no-codesign
```

Comment:
You're testing on a simulator below anyway. Build x86 to make things marginally faster.